Add step to ensure power is fully drained from components

winfred opened the pull request:
My thelio can get in this state even on a normal/safe shutdown sometimes. Has happened infrequently enough that I forget this crucial step each time haha

ahoneybun committed:
Update version numbers

n3m0-22 committed:
replace other instances of old name

n3m0-22 committed:
Merge branch 'master' into fix-upgrade-link

storyaddict committed:
Merge pull request #1074 from system76/update-dvd

Update version numbers

n3m0-22 committed:
Merge branch 'master' into fix-upgrade-link

n3m0-22 committed:
Merge branch 'master' into update-system76-power-daemon-name-in-fedora

burden committed:
update omnibar link (#1077)

* update omnibar link

* remove spooky theme

n3m0-22 committed:
Merge branch 'master' into update-system76-power-daemon-name-in-fedora

n3m0-22 committed:
Merge branch 'master' into fix-upgrade-link

n3m0-22 committed:
update-minimum-memory

This updates the minimum allocated memory to match the requirements listed below the iso download. Prompted by a user on Matermost who was unable to install Pop!_OS in VirtualBox with 2048 MB. https://chat.pop-os.org/pop-os/pl/oncw39g1eirgpmtgmay9fxeuah

burden committed:
update the contact us section (#1082)

jawadnassar committed:
Merge branch 'master' into removing-atom

jawadnassar committed:
replacing atom by Sublime Text

jawadnassar committed:
replacing atom by sublime Text

jawadnassar committed:
upload sublime editor image

leviport committed:
Merge pull request #1068 from jawadnassar/removing-atom

removed atom since it will be sunset soon

leviport committed:
Merge branch 'master' into fix-upgrade-link

n3m0-22 committed:
Merge branch 'master' into update-system76-power-daemon-name-in-fedora

n3m0-22 committed:
Merge branch 'master' into update-minimum-memory

leviport committed:
Merge pull request #1071 from system76/fix-upgrade-link

fix upgrade-older-releases link

leviport committed:
Merge branch 'master' into update-minimum-memory

leviport committed:
Merge pull request #1081 from system76/update-minimum-memory

update-minimum-memory

leviport committed:
Typofix: add missing / to NVMe fsck command

leviport committed:
Merge branch 'master' into update-system76-power-daemon-name-in-fedora

leviport committed:
Merge pull request #1073 from TopherIsSwell/update-system76-power-daemon-name-in-fedora

docfix: Updated power daemon service name for Fedora

n3m0-22 committed:
Merge branch 'master' into fsck-typo

leviport committed:
Merge pull request #1084 from system76/fsck-typo

Typofix: add missing / to NVMe fsck command

realKarthikNair committed:
Fix a minor confusion 

````nvme0n1p3```` won't necessarily be the Pop!_OS partition

jacobgkau committed:
fix: Grammar, whitespace, advice not specific to dual boots/EFI/NVMe

thomas-zimmerman committed:
Merge pull request #1085 from realKarthikNair/patch-1

bootloader: Note that partition numbers may be different in manual partitioning schemes

zachary-kaelan committed:
Add gaze17, oryp9, and oryp 10

Bringing it in line with what is listed here: https://github.com/pop-os/gnome-shell-extension-system76-power/blob/master/src/extension.ts

n3m0-22 committed:
gaze17

Added both gaze17 models

n3m0-22 committed:
revert gaze17 change

n3m0-22 committed:
remove oryp4-b sub-variant

ahoneybun committed:
Remove Docking Station article (#1087)

jacobgkau committed:
graphics-switch-ubuntu: Add additional models

13r0ck committed:
Update inscructions

clarification and some small changes

13r0ck committed:
lint fix

ahoneybun committed:
Update ar-sandbox.md

Fix lint issue

ahoneybun committed:
Update ar-sandbox.md

Fix lint

ahoneybun committed:
Update ar-sandbox.md

jasenmichael committed:
fix: protocol error

jasenmichael committed:
Merge branch 'master' into jh-fix-protocol-error

jasenmichael committed:
Merge pull request #1091 from system76/jh-fix-protocol-error

fix: protocol error

jasenmichael committed:
Merge branch 'master' into sarndbox

ahoneybun committed:
Update ar-sandbox.md

ahoneybun committed:
Update ar-sandbox.md

Fix typo

ahoneybun committed:
Update ar-sandbox.md

Fix typos

ahoneybun committed:
Update ar-sandbox.md

ahoneybun committed:
Update ar-sandbox.md

ahoneybun committed:
Update ar-sandbox.md

Socialhappiness committed:
Merge pull request #1090 from system76/sarndbox

Update instructions

Socialhappiness committed:
removed ubuntu dash bit and fixed a typo

ahoneybun committed:
Add missing models (#1075)

Ampersandstorm committed:
Update printer articles (#1063)

* Updated add-a-printer-pop

* Update add-a-printer-ubuntu

* Fix lint issues with ordered lists

* chore: Allow HTML links in order to enable new-tab links

Co-authored-by: Velocira <velocira@pop-os.localdomain>
Co-authored-by: Jacob Kauffmann <jacob@system76.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

Ampersandstorm committed:
Resolving ProtonVPN Killswitch Issues, Install Pop updates (#1066)

* Updated article instructions and screenshots

* First draft

* Add See Also section

* Lint issue

* Added Levi's 9-23-22 feedback

Co-authored-by: Velocira <velocira@pop-os.localdomain>
Co-authored-by: leviport <levi@system76.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Add Application menu text based on feedback from ticket

Ampersandstorm committed:
Add note about Gamehub compatibility - resolves issue #738 (#988)

* Add note about Gamehub compatibility

* fix linter issue, add image

* fixed typos

Co-authored-by: Nathaniel Warburton <55410594+storyaddict@users.noreply.github.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

storyaddict committed:
Strengthening Disclaimer (#1034)

* strengthening disclaimer

* revised laptop table and added secureboot note.

* Address feedback from QA

* Add models

* Add some edits from PR 973

* Address feedback from QA

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Randall White <N3M0-22@pm.me>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

garnerlogan65 committed:
revision for clarity (#969)

* revision for clarity

* Fixing lint issues

* Update bootloader.md

- Moved the explanation of a bootloader to the beginning of the 1st paragraph
- Split the run-on sentence in the "Important Note"

Co-authored-by: Nathaniel Warburton <55410594+storyaddict@users.noreply.github.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Zach Gangwer <4120010+gangwerz@users.noreply.github.com>
Co-authored-by: Randall White <N3M0-22@pm.me>

ahoneybun committed:
Add that Pop!_OS 22.04 LTS uses Tensorman  (#1062)

* Add that Pop!_OS 22.04 LTS uses Tensorman to address docs issue 494

* Address review and fix lint issue

* Fix misspelling, grammar issues, reword some awkward sentences, repositioned potentially confusing asterisk note

* Remove extraneous title words

Co-authored-by: Velocira <velocira@pop-os.localdomain>

ahoneybun committed:
Add missing install command (#1095)

emujoyce committed:
Fix link that is currently broken

thomas-zimmerman committed:
Merge pull request #1097 from emujoyce/patch-1

Fix link that is currently broken

ahoneybun committed:
Add Coreboot Configurator steps (#1060)

* Add Coreboot Configurator steps

* Update intel-me.md

* fix run-on sentence

Co-authored-by: Velocira <velocira@pop-os.localdomain>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

garrettjwilke committed:
Update ar-sandbox.md (#1099)

Changed file directory from:
/etc/SARndbox-1.6/BoxLayout.txt > /etc/SAR*/BoxLayout.txt

Software updates make this step difficult to follow, so adding a * glob will prevent this for any user encountering an error at this step.

jasenmichael committed:
fix: inaccurate last edited date

jasenmichael committed:
Merge pull request #1101 from system76/fix-last-edit-date

fix: inaccurate last edited date

garrettjwilke committed:
bad link to docs (#1104)

link to docs was bad link
changed to current link

mabhijithn committed:
Verify UEFI boot: Expected output correctly placed (#1105)

* Verify UEFI boot: Expected output correctly placed

UEFI boot verify command's expected output was put in the subsequent section incorrectly. This has been changed to its correct section.

* fix (CI): Remove duplicate blank line

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

acropup committed:
Uppercase Hash Algorithm for CertUtil

CertUtil in Windows 7 requires the HashAlgorithm option to be uppercase. In this case, SHA256 needs to be uppercase. In Windows 10, case does not matter. I have not verified with other versions.

leviport committed:
Merge pull request #1108 from acropup/patch-1

Uppercase Hash Algorithm for CertUtil

qin-guan committed:
fix: paru sync command (#1107)

* fix(system76-driver): paru sync command

* fix(system76-software): paru sync command

---------

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

leviport committed:
Add Krita to media production list

thomas-zimmerman committed:
Merge pull request #1111 from system76/media-krita

Add Krita to media production list

thomas-zimmerman committed:
Update log-files.md (#1112)

Fix a typo on file `lspci -vv` output is saved to.

ahoneybun committed:
Add NixOS configuration (#1113)

* Add NixOS configuration

* Update system76-software.md

Addressing feedback

---------

Co-authored-by: Aaron Honeycutt <aaronh@pop-os.localdomain>

jacobgkau committed:
boot-menu: Slightly reformat & add pang12 (#1117)

* boot-menu: Slightly reformat & add pang12

* fix: Allow manual HTML line breaks in Markdown linter

* boot-menu: Improve formatting for option names

ahoneybun committed:
Update touchpad disable (#1114)

* Update touchpad article and combine them

* Fix note

* Update note about the pang12 touchpad keycombo

* Address feedback

* fix: Correct grammar

* fix (touchpad): Update images, remove old images

* improv (touchpad): Update and simplify verbiage

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>
Co-authored-by: Jacob Kauffmann <jacob@system76.com>

ahoneybun committed:
Add Pipewire for Ubuntu (#1109)

* Add Ubuntu 22.10 as it uses Pipewire

* Update version

---------

Co-authored-by: leviport <levi@system76.com>

Commit:
Update content to match the Pop article a bit more

Commit:
Update verbiage based on feedback

thomas-zimmerman committed:
Merge pull request #1118 from system76/update-graphics-switch

Update content to match the Pop article a bit more

thomas-zimmerman committed:
Docker CUDA updates (#1119)

* Start work on updating CUDA but using Docker for 22.04+

* updates from test install on fresh install of Pop!OS 22.04

ran through the new instructions on getting versions of the CUDA library installed outside what is packaged with the Nvidia driver directly.
Updates to get docker.io installed before adding user to docker group
remove sudo on docker commands

* fix Bare URL used

* remove extra blank line

* replace then with than

* fix: Spelling, grammar, & phrasing in introduction paragraph

* cuda: Various formatting & content improvements

* fix(cuda): Correct commands for Docker tutorial

* remove "s" from 0_Introductions

* Fix space

* match directories in commands

* fix(cuda): Minor command adjustments

* fix(cuda): Run spell check

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Randall White <N3M0-22@pm.me>
Co-authored-by: Jacob Kauffmann <jacob@system76.com>

blulightspecial committed:
add macOS instructions to live-disk.md (#1120)

Commit:
Fix typo

leviport committed:
Merge pull request #1125 from system76/fix-software

Fix typo

thomas-zimmerman committed:
Add Launch Heavy (#1123)

* Add Launch Heavy

Add Launch Heavy and slight updates around firmware updates.

* Update launch-keyboard.md

* improv (launch): Include model codes in parenthesis

---------

Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: Jacob Kauffmann <jacob@system76.com>

thomas-zimmerman committed:
Update guides.md (#1124)

* Update guides.md

...with links to new tech-docs links

* Add new Thelio docs

* Update guides.md

Add Thelio Major R3 and fix other Thelio Major

* Update guides.md

Correct version for Thelio Major

* Add serw13 and addw3

* Fix lint issues

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: Randall White <N3M0-22@pm.me>

pparidans committed:
[FIX] kernelstub: wrong command for adding kernel options (#1122)

The `-o` option actually replaces the whole boot options' list passed to the kernel instead of adding to the configuration (if they aren't present already), which is done using `-a` (or the lengthier `--add-options`).

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Add models (#1126)

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

burden committed:
Update header with link to launch keyboard (#1128)

Commit:
Remove Apt preferences file

Commit:
Remove nomodeset option

Commit:
Ad 22.04 slideshow

Commit:
Remove old image and update disclaimer

thomas-zimmerman committed:
Merge pull request #1132 from system76/update-ubuntu-article

Update ubuntu article

thomas-zimmerman committed:
Updates to pop-recovery (#1121)

* Updates to pop-recovery

Customer feedback -- found that we were not clear that once you accessed drives, installer was not working for him with errors that are not explained/clear

* Fix lint issue

* Addres feedback

* Fix lint issue

* Adjust feedback and work on lint issue

* See if this passes

* Work on fixing lint issue

* Fix Markdown formatting & correct minor grammatical error

* Use proper name for refresh; don't state that refresh will lose data

* Reduce line-break spaces from 4 to 2

* Remove pre-COSMIC app menu reference

* Change first of two notes to use caution label

* Add a COSMIC-compatible alternative way to close the installer

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: Jacob Kauffmann <jacob@system76.com>

Commit:
Address issues

thomas-zimmerman committed:
Update difference-between-pop-ubuntu.md

Updated language on recovery
Fix all Pop!_OS >> Pop!\_OS

Commit:
Address feedback

Commit:
Merge branch 'update-differences-article' of github.com:system76/docs into update-differences-article

Commit:
Address feedback

Commit:
Fix typo

Commit:
Address feedback/3

Commit:
Fix typo

Commit:
Fix lint issue

Commit:
Work on issue 971

Commit:
Revert change

Commit:
Add about using the Refresh option

leviport committed:
Merge pull request #1136 from system76/update-differences-article

Address issues

ahoneybun committed:
Merge branch 'master' into promote-recovery

thomas-zimmerman committed:
document manual steps to update launch_2 and launch_heavy_1 keyboards (#1140)

* document manual steps to update launch_2 and launch_heavy_1 keyboards

* Fix lint issues

* Fix grammar issues

* Add config export section, and update structure and wording

* Add note about the firmware version that this article applies to

* Remove verification section and verify with configurator instead

* Add "Update firmware" section

* Capitalize Launch Keyboard

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: leviport <levi@system76.com>

ahoneybun committed:
Add about main battery (#1142)

* Add about main battery

* Update power-on-failure-laptop.md

Update based on feedback

* Update power-on-failure-laptop.md

Address feedback

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

Commit:
Add missing models

jacobgkau committed:
fix (lint): Remove trailing space

thomas-zimmerman committed:
Merge pull request #1147 from system76/update-tech-docs

Add `bonw15`, `galp7`, `gaze18` models

ahoneybun committed:
Add missing `addw3`, `gaze18`, `oryp11` models (#1148)

* Add missing models

* Add addw3 and oryp11

* Add oryp11 to ubuntu article

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

burden committed:
update headerbar with components link (#1149)

ahoneybun committed:
Add models and update text (#1152)

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

ahoneybun committed:
Merge branch 'master' into promote-recovery

burden committed:
update omnibar with back to school sale details (#1156)

thomas-zimmerman committed:
Update Yubikey article (#1054)

* updates to yubikey setup. Add some screen shots.

* Fix lint issues

* Fix lint

* Fix last lint issue

* Address feedback

* Fix lint issue

* Address feedback

* Update based on feedback

---------

Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

thomas-zimmerman committed:
Update to allow AppImage to work on Ubuntu (#1153)

Ubuntu requires `libfuse2` to be installed for AppImage to work.

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

Commit:
Move article to archived

Commit:
Move article to archived

Commit:
Set the article to hidden

Commit:
Set the article to hidden

ahoneybun committed:
Updating VPN article and moving OpenVPN article as it is not needed (#1139)

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

ahoneybun committed:
Merge branch 'master' into promote-recovery

jacobgkau committed:
Improve wording & address feedback about home dir formatting

thomas-zimmerman committed:
Merge pull request #1138 from system76/promote-recovery

Work on Issue 971

Commit:
Merge branch 'issue-1150' of github.com:system76/docs into issue-1150

ahoneybun committed:
Merge branch 'master' into issue-1150

Commit:
Fix build issue

Commit:
Merge branch 'issue-1150' of github.com:system76/docs into issue-1150

Commit:
Fix lint issue

thomas-zimmerman committed:
Merge pull request #1157 from system76/issue-1150

Move roadmap article to archived

leviport committed:
Add note about new 90% default start threshold (#1163)

burden committed:
Update footer (#1162)

Socialhappiness committed:
adding images for new support article about elarging desktop

Socialhappiness committed:
add images for desktop zoom article

ahoneybun committed:
Add links to pang13

ahoneybun committed:
Work on adding the advanced luks article to the password article to combine them fixing issue 1161

ahoneybun committed:
address issue 1164

ahoneybun committed:
Update guides.md

thomas-zimmerman committed:
Merge pull request #1169 from system76/add-pang13

Add link to pang13

ahoneybun committed:
Merge branch 'master' into pang12-flex

Commit:
Fix lint issue

ahoneybun committed:
Merge branch 'master' into update-passwords

ahoneybun committed:
Update Other OS steps based on feedback (#1154)

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

ahoneybun committed:
Update package manager (#1137)

* Update content

* Add commands

* Fix lint issue

* Update wording to directly mention files that will be created

* Pop!_Shop > Pop Shop
Update to not mention Pop Shop but Ubuntu Store -- Software & Updates

Update both for Apt > APT

* Fix spelling and formatting

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Thomas Zimmerman <thomas@system76.com>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

winfred committed:
Add step to ensure power is fully drained from components

My thelio can get in this state even on a normal/safe shutdown sometimes. Has happened infrequently enough that I forget this crucial step each time haha

jacobgkau commented:
This was discussed somewhat thoroughly in https://github.com/system76/docs/pull/1053.

This PR is better since it doesn't unnecessary state to repeat the procedure multiple times. Although it still lists the power switch being on before it suggests unplugging the system, that order of troubleshooting makes sense based on how commonly each step might be needed.

I do want to clarify, when you say `but you see lights on`, is that the front power button, or lights inside of the machine? If it's the former, then we might want to move this underneath step 5. (Or we could make this more general by removing that note.)

burden committed:
update @system76/components package (#1170)

* update @system76/components package

* update link

* spelled right

jonknapp committed:
Update wireless.md (#1143)

Add info about disabling Windows "Fast Startup" which was the cause of my missing wifi card after a driver update from the Windows side.

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Update driver software (#1127)

* Update order and switch to git versions

* Add io package

* Fix order

* Update Fedora steps

* Add content and reorder it

* Update system76-software.md

extended update around gnome-extension that it does not work on newer GNOME releases.

* Remove Paru line in Fedora steps

* Address feedback

* Update folder names

* Replace system76-firmware-daemon with base system76-firmware package

* Update repo

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

ahoneybun committed:
Update customize gnome (#1135)

* Update Pop!_OS spelling and add notes

* Remove Gnomenu as it does not support GNOME 42

* Remove Extension Update Notifier as it is no longer supported and it is built into extensions.gnome.org now

* Fix lint issue

* Address feedback

* Update customize-gnome.md

extend note on Dash-to-Dock and Cosmic-Dock

* Address feedback

* Fix Pop spelling

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

burden committed:
fix builds for node 18+ (#1174)

* support for node 18

* update @system76/design

Commit:
Update image

ahoneybun committed:
Merge branch 'master' into pang12-flex

thomas-zimmerman committed:
Merge pull request #1167 from system76/pang12-flex

address issue 1164

ahoneybun committed:
Add section for tech docs and guides (#1145)

* Add section for tech docs and guides

* Add keywords for discovery

* Add section in article to guides

* Remove section

* Remove keywords from incorrect article

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

ahoneybun committed:
New Fan Noise article (#1171)

* Init work on Fan Noise article

* Fix lint issues

* Fix lint issues

* Update fan-noise.md

Worked on the language

* Update fan-noise.md

...and additional language updates

* add image and fix lint issue

* Address feedback

* Update list for newer and older models

* Address feedback

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

ahoneybun committed:
Merge branch 'master' into update-passwords

ahoneybun committed:
Update linux gaming (#1175)

* Update image to new Steam UI

* Update search screenshot and activities to application for 22.04

* Update screenshots

* Update wording

* Move using steam section

* remove note about GameHub as it works now.

* Update linux-gaming.md

fix feedback

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

thomas-zimmerman committed:
Update password.md

updates to the LUKS commands.

Commit:
Fix lint issue

ahoneybun committed:
Merge branch 'master' into update-passwords

thomas-zimmerman committed:
Add note to use launch cables for firmware updates (#1177)

* Add note to use launch cables for firmware updates

Update launch_2-firmware-update.md

* Address feedback

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

ahoneybun committed:
Merge branch 'master' into update-passwords

ahoneybun committed:
Update dock article (#1178)

* Update images and renaming article

* Remove Ubuntu 16.04 section as it is no longer supported

* Add back Dash to Dock

* Add content

* Update images and remove old ones

* Rename article to fix link

* Fix lint issue

* Fix lint issue and address feedback

* Remove Dash to Dock installation and instead use dconf editor

* Correct position of the default dock for Ubuntu and Pop!_OS and adjust other content

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Thomas Zimmerman <thomas@system76.com>

thomas-zimmerman committed:
Log in from live disk (#1180)

* Update login-from-live-disk.md

Added context to encryption section

* Add some additional comments on what the chroot commands are doing and fixing lint issues

* Address feedback

---------

Co-authored-by: Sara Mildenstein <115432964+sararocks@users.noreply.github.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Fix shortcuts and address feedback

ahoneybun committed:
Change table to command to check hardware (#1182)

burden committed:
remove gear up promo (#1188)

* remove gear up promo

* update omnibar

Spongecade committed:
Update Minecraft wiki link to new domain (#1187)

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

mapachurro committed:
Remove duplicate instruction (#1189)

* Remove duplicate instruction

The instruction regarding taking a picture or screenshot of the instructions showed up twice. I left the later one, as I feel it provided better emphasis to an important point.

* Update system-firmware.md

* Update system-firmware.md

* Update system-firmware.md

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Add Boot Menu from the thelio-r3 board (#1183)

Co-authored-by: Francis Smith <fs1995@ymail.com>

ahoneybun committed:
Add meer7 and darp9 (#1185)

* Add meer7

* Add darp9, meer7 and fix lint issue

---------

Co-authored-by: Francis Smith <fs1995@ymail.com>

ahoneybun committed:
Update screenshots and content for 22.04 (#1181)

* Update screenshots and content for 22.04

* Remove Beta line and fix grammar

---------

Co-authored-by: Francis Smith <fs1995@ymail.com>

ahoneybun committed:
Add note about the pang12/13 switch (#1179)

* Add note about the pang12/13 switch

* webcam: Reword article to better incorporate pang12/13 switch

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Jacob Kauffmann <jacob@system76.com>
Co-authored-by: Francis Smith <fs1995@ymail.com>

Socialhappiness committed:
Create Desktop_zoom.md (#1165)

* Create Desktop_zoom.md

adding an article specifically for making desktop bigger

* work on fixing lint issues

* Rename file

* More lint fixes

* Fix lint issues

* Fix numbered list issue

* Another possible lint fix

* Update desktop-zoom.md

* Add lint rule to keep numbered list with images working

* Update desktop-zoom.md

* Add images

* Fix image links

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

ahoneybun committed:
Update ubuntu articles (#1176)

* Add note about GNOME Shell extension for newer then Ubuntu 22.04

* Link to boot menu article for up to date steps and address feedback

* Address feedback about note

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Levi Portenier <levi@system76.com>
Co-authored-by: Francis Smith <fs1995@ymail.com>

JoseJacsan committed:
Added Shotcut video editor description in list (#1191)

* Added Shotcut video editor description in list

* Added spelling error correction

* Add additional descriptors from the project's homepage

* Update media-production-on-pop.md

Fix lint issue

* fix (lint): Remove double line break at end of file

---------

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

adamdicarlo0 committed:
Fix misplaced word and use "back up" (verb) instead of "backup" (#1193)

burden committed:
Update omnibar with link to Tuxmas page (#1194)

burden committed:
tuxmas sale week 2, update components lib + omnibar copy (#1195)

burden committed:
tuxmas sale week 3 (#1196)

garrettjwilke committed:
bash basics tutorial (#1197)

* bash basics tutorial

* Address feedback

* Address feedback

* Rewrite adding functionality section

---------

Co-authored-by: Garrett Wilke <garrett@system76.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: leviport <levi@system76.com>

ahoneybun committed:
Update images and comment out CLI option as does not support that in v4 yet

Fix lint

thomas-zimmerman committed:
Merge pull request #1200 from system76/update-protonvpn-killswitch

Update images and comment out CLI option

thomas-zimmerman committed:
Merge branch 'master' into update-passwords

thomas-zimmerman committed:
Merge pull request #1166 from system76/update-passwords

Work on adding the advanced luks article to the password article

memachado committed:
Update desktop-environment.md (#1190)

* Update desktop-environment.md

This addition offers users a more comprehensive solution for removing KDE and its associated packages. This enhancement aims to provide clearer and more user-friendly instructions for managing desktop environments on Pop!_OS.

* Split regex from normal autoremove and add explanation

---------

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>

macifell committed:
tuxmas phase 4

macifell committed:
Merge pull request #1205 from system76/tuxmas4

tuxmas phase 4

ahoneybun committed:
Add meer8 to guide and windows articles

bbarker committed:
Update difference-between-pop-ubuntu.md (#1209)

Fix typo in keyboard shortcut for switching workspaces

macifell committed:
Remove tuxmas

macifell committed:
Merge branch 'master' into remove-tuxmas

macifell committed:
Merge pull request #1211 from system76/remove-tuxmas

Remove tuxmas

seansawyer committed:
Update guides.md to include Lemur Pro 12 (lemp12) tech doc (#1212)

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

PeterGrand committed:
Use recursive binding before chroot (#1204)

By using -R instead of -B, we ensure that the nested mount of /sys/firmware/efi/efivars/ becomes mounted in our target filesystem, which allows us to do various boot repairs with an EFI system.

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

samcodee committed:
Update article about Differences between Pop!_OS and Ubuntu (#1203)

* Add updated intel nvidia

* Fix typo: "Backported"

* Add new image for download screen

also added info about rpi 4

* Rename intel_nvidia.png to intel-nvidia-rpi.png

* Delete old image

---------

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Add pang13 and pang14 to missing spots in articles (#1213)

ahoneybun committed:
Add nix-shell file to use nix for testing

ahoneybun committed:
Update shell.nix

remove shellHooks

ahoneybun committed:
add launch_3 and launch_heavy_3 tech doc links

jklgrasso committed:
Update hardware-failure.md

Changed the gpu-burn example to use tc instead of d to create more of a workload when testing.

jklgrasso committed:
Update hardware-failure.md

Adjusted for RTX GPUs to use -tc and removed the -d switch for GTX GPUS.

thomas-zimmerman committed:
Merge pull request #1221 from system76/gpu-burn-d-to-tc

Update hardware-failure.md

ahoneybun committed:
Fix typo for Packages

thomas-zimmerman committed:
Merge pull request #1207 from system76/add-meer8

Add meer8

thomas-zimmerman committed:
Merge branch 'master' into add-nix-shell

thomas-zimmerman committed:
Merge branch 'master' into update-launch

thomas-zimmerman committed:
Merge pull request #1219 from system76/update-launch

add launch_3 and launch_heavy_3 tech doc links

thomas-zimmerman committed:
Merge branch 'master' into add-nix-shell

thomas-zimmerman committed:
Merge pull request #1216 from system76/add-nix-shell

Add nix-shell file to use nix for testing

ahoneybun committed:
Clean up and remove old install Ubuntu images for install-ubuntu article

ahoneybun committed:
Update/Move content and update screenshots

ahoneybun committed:
Add more images

ahoneybun committed:
Add Software Updater screenshots

ahoneybun committed:
update: clean up the windows table and add missing systems (#1214)

* update: clean up the windows table and add missing systems

* fix: typo and lint issue

* Update windows.md

Remove left over from merge conflict

---------

Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

ahoneybun committed:
Update deploy.yml (#1225)

* Update deploy.yml

* Update ci.yml

* Update deploy.yml

macifell committed:
Revert "Update deploy.yml (#1225)"

This reverts commit 5427233722b0a763646de0ca615ef10c81c59432.

thomas-zimmerman committed:
Merge pull request #1227 from system76/revert-1225-update-workflow

Revert "Update deploy.yml"

ahoneybun committed:
Finish up article with initial setup screenshots

leviport committed:
Merge branch 'master' into update-ubuntu-24.04

ahoneybun committed:
Update Ubuntu supported release and links for making USB while removing DVD reference

ahoneybun committed:
Address the rest of the feedback minus two from the review

ahoneybun committed:
Update GRUB and LUKS/Plymouth screenshots

ahoneybun committed:
Cleaned up remaining fixes in the review

ahoneybun committed:
Updates the wording about third-party software and media support selection in the installer

ahoneybun committed:
Address feedback

jacobgkau committed:
fix: Grammar & typos

jacobgkau committed:
fix: Additional ordering, grammar, and consistency tweaks

thomas-zimmerman committed:
Merge pull request #1228 from system76/update-ubuntu-24.04

Update install-ubuntu article for 24.04

ryanbaekr committed:
Add gnome-shell-extensions to the packages needed for Ubuntu (#1232)

* Add gnome-shell-extensions to the packages needed for Ubuntu

Ubuntu 24.04 doesn't ship with gnome-shell-extensions

* Update pop-shell.md

* improv: List reason for installing extension-prefs package

---------

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

taivlam committed:
Fix typo in launch-keyboard.md (#1234)

Use "its" (possessive), instead of "it's" (contraction)

taivlam committed:
Fix typo in launch_2-firmware-update.md (#1233)

Remove extra "for"

Co-authored-by: Levi Portenier <levi@system76.com>

jacobgkau committed:
Add ROCm article (#1231)

ahoneybun committed:
Merge branch 'master' into fix-typo

Commit:
Merge pull request #1223 from system76/fix-typo

Fix typo for Packages

ahoneybun committed:
Update model list and remove unsupported OSes (#1224)

ahoneybun committed:
Redo bootloader (#1198)

* Reorder article and update sections

* Update notes about BIOS installs

* Fix header for encrypted disk

* Add Pop!_OS GRUB steps and fix lint issue

* Address feedback

* Update bootloader.md

Removed > on line 85 to (hopefully) fix markdown failure.

---------

Co-authored-by: brandonbaez7 <160044194+brandonbaez7@users.noreply.github.com>

jacobgkau committed:
Add recent tech docs models & link to tech docs homepage (#1237)

thomas-zimmerman committed:
add note for EC update (#1239)

* add note for EC update

Firmware updates that include EC firmware updates require a full power cycle to complete. Added a note suggesting either AC power unplug/plug or holding power button for 15 seconds.

* Update system-firmware.md

..and fix language.

spaceoden committed:
Add note about need to disable ppd on NixOS (#1240)

* Add Nixos power note to system76-software.md

ppd (enabled by default on Gnome NixOS installs) must be disabled for system76-power to start. Include that info in docs.

* Update system76-software.md

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>

ws909 committed:
Update the steps to generate a log file in freezing-behavior.md (#1241)

* Update the steps to generate a log file in freezing-behavior.md

The existing explanation was for a previous version of the system.

* Update freezing-behavior.md

Readd the log file creation in the System76 Driver in addition to in the Settings application

* Update freezing-behavior.md

fix Pop!_OS

* Update freezing-behavior.md

* Update freezing-behavior.md

fix lint

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

UrielCh committed:
Update cuda.md, replace cuda 12.2 with 12.4 (#1238)

* Update cuda.md, replace cuda 12.2 by 12.5

Cuda 12.2 is deprecated, and display nasty warning.

the driver version 555 is currently 6 day old.

the driver version 550 was built on top of cuda 12.4.

* replace cuda 12.5 by 12.4

---------

Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>
